pandas/core/series.py

```diff
@@ -301,6 +301,20 @@ def _update_inplace(self, result, **kwargs):
         # we want to call the generic version and not the IndexOpsMixin
         return generic.NDFrame._update_inplace(self, result, **kwargs)
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        if value is not None:
+            if com.is_hashable(value):
```

Comment:
Ok, but how better add object.__setattr__(self, '_name', value) to this code?
object.__setattr__(self, '_name', None) only if name attribute not added.